examples/examples-gpu/nyc-taxi/rf-rapids-dask.ipynb

```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p><code>random forest OR need a refresher</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>lol oh right, thank you</p>

Reply:
<p>fixed in <a href="https://github.com/saturncloud/examples/pull/58/commits/f092304d317931e1a0f02be86178a58ec58b2bbf" target="_blank">https://github.com/saturncloud/examples/pull/58/commits/f092304d317931e1a0f02be86178a58ec58b2bbf</a></p>